assets/js/shortcode-insert.js

```diff
@@ -99,7 +99,7 @@ jQuery(function($){
 
 	});
 	
-	if( typeof wp !== 'undefined' && typeof wp.media !== 'undefined' ){
+	if( typeof wp !== 'undefined' && typeof wp.media !== 'undefined' && $("#caldera-forms-form-insert").length != 0 ){
```

Comment:
I'm merging as is, but I do wonder if this whole thing file should be wrapped in a check `if( $("#caldera-forms-form-insert").length != 0 )` But let's not risk breaking shit when this whole thing is about to be backwards compat scope after WP 5.0

Reply:
@New0 ^^